.circleci/config.yml

```diff
@@ -122,3 +131,6 @@ workflows:
       - test_27
       - test_35
       - test_36
+      - test_37
+      - test_38
+      - test_39
```

Comment:
I'm not really familiar with your CI obviously so this might not make sense. Let me know if I should revert this.